clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -246,7 +246,7 @@ public class VenicePushJob implements AutoCloseable {
    * 3. Negative enums are error scenarios (Can be user or system errors)
    */
   public enum PushJobCheckpoints {
-    INITIALIZE_PUSH_JOB(0), NEW_VERSION_CREATED(1), START_MAP_REDUCE_JOB(2), MAP_REDUCE_JOB_COMPLETED(3),
+    INITIALIZE_PUSH_JOB(0), NEW_VERSION_CREATED(1), START_DATA_WRITER_JOB(2), DATA_WRITER_JOB_COMPLETED(3),
```

Comment:
It is safe to rename them since we only use persist the checkpoint values (and not the names) in the push status system store. We need to udpate the names in the consumers of this report separately